lms/djangoapps/completion/utils.py

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
I really don't think this belongs in this dir, but it was difficult for me to find a spot where it DID belong. I'll have to send this to the marketing site via API (it's looking like) so I should probably move this.